cmake/SundialsSetupCompilers.cmake

```diff
@@ -476,20 +476,42 @@ endforeach()
 # Configure clang-tidy for linting
 # ===============================================================
 
+set(SUNDIALS_DEV_CLANG_TIDY_DIR ${CMAKE_BINARY_DIR}/clang-tidy/)
+
 if(SUNDIALS_DEV_CLANG_TIDY)
-  make_directory(${CMAKE_BINARY_DIR}/clang-tidy)
+  find_program(CLANG_TIDY_PATH NAMES clang-tidy)
+  if(NOT CLANG_TIDY_PATH)
+      message(FATAL_ERROR "Could not find the program clang-tidy")
+  endif()
+  message("Found IWYU: ${CLANG_TIDY_PATH}")
+  make_directory(${SUNDIALS_DEV_IWYU_DIR})
+  make_directory(${SUNDIALS_DEV_CLANG_TIDY_DIR})
   if(SUNDIALS_DEV_CLANG_TIDY_FIX_ERRORS)
```

Comment:
Should this be added as an advanced option?

Reply:
There is no reason it should need to be changed, so I think its best to leave it out for now to reduce the number of options that display for developers.